spring-xd-rest-client/src/main/java/org/springframework/xd/rest/client/ResourceOperations.java

```diff
@@ -50,4 +50,10 @@
 	 */
 	public void destroyAll();
 
+	/**
+	 * Delete a RabbitMQ queue.
+	 */
+	public void deleteQueue(String name, Map<String, String> properties);
```

Comment:
`deleteQueue` is not really all it does; it deletes all the queues (and DLQs) for the stream as well as any exchanges that were created for taps.

Suggest `clean`, `cleanBusResources` or similar.